packages/ogmios/src/CardanoNode/OgmiosCardanoNode.ts

```diff
@@ -65,7 +65,12 @@ export class OgmiosCardanoNode extends RunnableModule implements CardanoNode {
       this.#logger.info('Getting system start');
       return await this.#stateQueryClient.systemStart();
     } catch (error) {
-      throw CardanoNodeUtil.asCardanoNodeError(error) || new CardanoNodeErrors.UnknownCardanoNodeError(error);
+      const domainError =
+        CardanoNodeUtil.asCardanoNodeError(error) || new CardanoNodeErrors.UnknownCardanoNodeError(error);
```

Comment:
Maybe let's keep the stack trace for UnknownCardanoNodeError - it could be useful